Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -22,28 +22,90 @@ public final class GDSInformationViewController: BaseViewController, TitledViewC
         fatalError("init(coder:) has not been implemented")
     }
     
-    @IBOutlet private var informationImage: UIImageView! {
-        didSet {
-            let font = UIFont(style: .largeTitle, weight: viewModel.imageWeight ?? .semibold)
-            let configuration = UIImage.SymbolConfiguration(font: font, scale: .large)
-            informationImage.image = UIImage(systemName: viewModel.image, withConfiguration: configuration)
-            informationImage.tintColor = viewModel.imageColour ?? .gdsPrimary
-            informationImage.accessibilityIdentifier = "information-image"
+    // swiftlint:disable private_outlet
+    @IBOutlet internal var bottomStack: UIStackView!
```

Comment:
@benmckillop-dd I made this outlet internal for testing. I was thinking the best test the footnote functionality is to actually change the bottomStack height - so we know for all device sizes its moving at the correct size. To stop swiftlint I put comments around it and sonar has [created an issue](https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=109&id=di-mobile-ios-common&open=AZO2tpQSEEsoNy0fy3ce). Before I mark this as accepted - just wanted to check if this is okay to mark internal :)

Reply:
Have you tried the `findChildView(byAccessibilityIdentifier:)` method instead? That's usually how we avoid making things internal. Otherwise `private(set)` would be better than `internal`.

Reply:
^ echo Oli's comment - findChildView would be the way forward usually - do let me know if you have issue though and we can pair on it

Reply:
thank you both! haha it was a really simple fix just had to add an accessibility identifier for bottom stack  :)